linux/scripts/dist.sh

```diff
@@ -16,36 +16,36 @@ THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BA
 ## END STANDARD BUILD SCRIPT INCLUDE
 
 BASEDIR=$(pwd)
-autotool_projects="kmflcomp libkmfl ibus-kmfl ibus-keyman"
-extra_projects="keyboardprocessor keyman-config"
+legacy_projects="kmflcomp libkmfl ibus-kmfl"
+extra_projects="keyman"
 
 if [ "$1" == "origdist" ]; then
     if [ "$2" != "" ]; then
         echo "$2"
-        if [ ! -d "$2" ]; then
+        if [ ! -d "$2" ] && [ "$2" != "keyman" ]; then
             echo "project $2 does not exist"
             exit 1
         fi
-        if [ "$2" == "keyman-config" ] || [ "$2" == "keyboardprocessor" ]; then
-            autotool_projects=""
+        if [ "$2" == "keyman" ]; then
+            legacy_projects=""
             extra_projects="$2"
         else
-            autotool_projects="$2"
+            legacy_projects="$2"
             extra_projects=""
         fi
     fi
 else
     if [ "$1" != "" ]; then
         echo "$1"
-        if [ ! -d "$1" ]; then
+        if [ ! -d "$1" ] && [ "$1" != "keyman" ]; then
             echo "project $1 does not exist"
             exit 1
         fi
-        if [ "$1" == "keyman-config" ] || [ "$1" == "keyboardprocessor" ]; then
-            autotool_projects=""
+        if [ "$1" == "keyman" ]; then
+            legacy_projects=""
             extra_projects="$1"
         else
-            autotool_projects="$1"
+            legacy_projects="$1"
             extra_projects=""
         fi
     fi
```

Comment:
This looks like the if/else conditions are essentially identical except for the parameter position. Could you consolidate?

Reply:
Consolidate the consolidation? Done.